uwblueprint/planet-read
# Auto resize EditableCell height based on content

## Notion ticket link



[Auto resize Editable height based on content](https://www.notion.so/uwblueprintexecs/Auto-resize-Editable-height-based-on-content-d49922a2853247c5aadb627432b5314e)



## Implementation description

- Set the heights of the editable. to always be the maximum of the two columns
- the height for EditableCell is calculated using the [useRef hook](https://reactjs.org/docs/hooks-reference.html#useref) 
- needed to make some css changes to TranslationTable and a couple other components 
- (this ended up being a lot more code change than I originally anticipated)



## Steps to test

1. open any story in translation
2. type in the boxes ! the cells in any row should always be the same height, and the text in the editable cell should always be visible with no scroll 
3. go to the review page, make sure that all the text is visible and the cells are the same height 
4. make sure styling for other components didn't get messed up 



## What should reviewers focus on?

- EditableCell.tsx 

## Checklist
